intelmq/lib/upgrades.py

```diff
@@ -824,6 +835,21 @@ def v310_feed_changes(configuration, harmonization, dry_run, **kwargs):
     if found_dshield_domain:
         messages.append('A discontinued feed "DShield Suspicious Domain" has been found '
                         'as bot %s.' % ', '.join(sorted(found_dshield_domain)))
+
+    if found_abusech_feodotracker_csv:
+        messages.append('A discontinued feed "Abuse.ch Feodo Tracker IPs" has been found'
+                        'as bot %s.\nPlease manually replace with the feed'
+                        '"Abuse.ch Feodo Tracker".' % ', '.join(sorted(found_abusech_feodotracker_csv)))
+
+    if found_abusech_feodotracker_browse:
+        messages.append('A discontinued feed "Abuse.ch Feodo Tracker Browse" has been found'
+                        'as bot %s.\nPlease manually replace with the feed'
+                        '"Abuse.ch Feodo Tracker".' % ', '.join(sorted(found_abusech_feodotracker_browse)))
+
+    if found_abusech_removed_parsers:
+        messages.append('A discontinued bot module has been found'
+                        'as bot %s.' % ', '.join(sorted(found_abusech_removed_parsers)))
+
```

Comment:
Why not changing the config directly? Replacing the module and adapting the `http_url` can be fully automated.

Reply:
I made the explanation into an issue #2297.

Reply:
If you insist on changing the config directly I will do it.

However as a user I prefer only being notified that I need to change to the config. I like to understand the necessary changes even if it requires more (manual) effort. That's why I implemented the upgrades this way.

Reply:
> Either as "run command xyz to automatically update documentation"

Originally that was the purpose of the `intelmqctl upgrade-config` command.